pkg/meta/redis.go

```diff
@@ -164,7 +165,38 @@ func newRedisMeta(url string, conf *Config) (Meta, error) {
 	}
 
 	m.checkServerConfig()
-	return m, nil
+	m.root = 1
+	m.root, err = lookupSubdir(m, conf.Subdir)
+	return m, err
+}
+
+func lookupSubdir(m Meta, subdir string) (Ino, error) {
```

Comment:
move this function to interface.go?

Reply:
Let's keep it here, we may put these together as  `struct defaultMeta{}`